Frontend/src/app/views/dashboard/time/time.component.scss

```diff
@@ -16,7 +16,9 @@ button {
   gap: 10px;
   padding-bottom: 50px;
   justify-content: space-between;
-  h6 {
+  text-align: center;

```

Comment:
I don't see a difference with or without this rule, what did this change?

Reply:
This makes sure the date in mobile view is centered between the arrow buttons.